msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-meeting-stage.md

```diff
@@ -0,0 +1,196 @@
+
+#### Scope
+
+The scope defines who can access the apps.
+
+* `groupchat` scope makes your app available in group scope and enables app to be added in a call or meeting (scheduled private meeting/ instant meetings).
+
+* `team` scope makes your app available in team scope and enables your app to be added in team or channel or scheduled channel meeting.
+
+#### Context
+
+The `context` property determines if the app is available in specific view after installation and configuration. Following are the values for the `context` property from which you can use all or some of the values:
+
+|Value|Description|
+|---|---|
+| **channelTab** | A tab in the header of a team channel. |
+| **privateChatTab** | A tab in the header of a group chat between a set of users, not in the context of a team or meeting. |
+| **meetingChatTab** | A tab in the header of a group chat between a set of users for a scheduled meeting. You can specify either **meetingChatTab** or **meetingDetailsTab** to ensure the apps work in mobile. |
+| **meetingDetailsTab** | A tab in the header of the meeting details view of the calendar. You can specify either **meetingChatTab** or **meetingDetailsTab** to ensure the apps work in mobile. |
+| **meetingSidePanel** | An in-meeting panel opened through the unified bar (U-bar). |
+| **meetingStage** | An app from the `meetingSidePanel` can be shared to the meeting stage. You can't use this app either on mobile or Teams room clients. |
+
+#### Configure tab app for a meeting
+
+Apps in meetings can use the following contexts: `meetingChatTab`, `meetingDetailsTab`, `meetingSidePanel` and `meetingStage`. Once a meeting participant installs an app and configures the tab in meeting, all targeted other contexts of the app for the given meeting will start rendering the tab.
+
+The following code snippet is an example of a configurable tab used in an app for Teams meetings:
+
+```json
+
+"configurableTabs": [
+    {
+      "configurationUrl": "https://contoso.com/teamstab/configure",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "team",
+        "groupchat"
+      ],
+      "context":[
+        "channelTab",
+        "privateChatTab",
+        "meetingChatTab",
+        "meetingDetailsTab",
+        "meetingSidePanel",
+        "meetingStage"
+     ]
+    }
+  ]
+```
+
+---
+
+#### Other examples
+
+Default context for tabs (if not specified) is:  
+
+```json
+"context":[ 
+  "channelTab", 
+  "privateChatTab", 
+  "meetingChatTab", 
+  "meetingDetailsTab" 
+     ] 
+```
+
+To prevent an app from showing in non-meeting group chats, you would set the following context on their tab:
+
+```json
+"context":[ 
+  "meetingSidePanel", 
+  "meetingChatTab", 
+  "meetingDetailsTab" 
+     ] 
+```
+
+To target just the in-meeting side panel experience:  
+
+```json
+"context":[ 
+  "meetingSidePanel” 
```

Comment:
Nit: use straight quotes, not curly (the closing double quote is curly, hence the syntax warning)